CMake/cdat_modules_extra/python_configure_step.cmake.in

```diff
@@ -29,7 +29,7 @@ if(APPLE)
 endif()
 
 if(APPLE)
-  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
+  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --with-system-expat --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
```

Comment:
This is probably safe.  AFAIK, `libexpat` is a Mac system library.  Like @durack1, I have no idea why using python's `libexpat` would fail on travis.